tracks/python/exercises/raindrops/mentoring.md

```diff
@@ -26,12 +26,12 @@ def raindrops(val: int) -> str:
     return result
 ```
 
-#### Solutions That Need Work
+### Solutions That Need Work
```

Comment:
```suggestion
### Common Suggestions
```

It think this actually works in terms of this document :)

Reply:
"Solutions that need work" suggests to the mentor that the student's solution though correct, still needs improvement and optimization. 
"Common suggestions" seems more ambiguous. 
But I understand why you're saying that, and I think the Talking Points section covers that. :)

Reply:
Yes, I'm just trying it to match the other notes in the track